backend/src/modules/github/controller.ts

```diff
@@ -0,0 +1,46 @@
+import { Request, Response } from 'express';
+import { HttpError, errorWrapper, getParams, httpStatusCode } from '../../utils';
+import { getOpenIssuesParamsSchema } from './schemas';
+
+/**
+ * Use github api to lookup open issues for a repo.
+ * @method GET
+ * @route /github/issues/open/:repo
+ */
+export const getOpenIssues = errorWrapper(async (req: Request, res: Response) => {
```

Comment:
Should we change this to `getIssues` and add a query parameter for `state` which we can just pass `open`?

Reply:
@TaylorFries @SDToews

Reply:
that or if we wanted to get closed issues we could add getClosedIssues in the future, OR in the future we can add a getClosedIssues and getAll which would call both and then merge them.